### DISCUSSION TITLE
How to add multiple classnames?

### DISCUSSION THREAD

**Discussion by alexradulescu** [mayank99/ecsstatic#22] [Q&A]
Hi,

Probably stupid question but imagine the following scenario:
- I have a generic button classname, that I share among multiple components
- I also have a special override to that one in a component
- I also get a css classname from the parent component?

Something like...
```
import { css } from '@acab/ecsstatic';
import { defaultButton } from './shared';

export const Button = ({className, ...props}) => {
  return <button {...props} className={???} />; // I want all 3, `defaultButton`, `specialButton` and whatever classname comes from parent component
};

const specialButton = css`
  ...
`;

**Comment by mayank99** [comment]
It's just string concatenation. You can do something like this:

```jsx
className={`${defaultButton} ${specialButton} ${props.className}`}
```

or in a more fancy way:

```jsx
className={[defaultButton, specialButton, props.className].filter(Boolean).join(" ")}
```

If you need nicer syntax or additional customization, you can use a small runtime utility like [`clsx`](https://github.com/lukeed/clsx) or [`classnames`](https://github.com/JedWatson/classnames).
